optax/_src/alias.py

```diff
@@ -1488,7 +1488,63 @@ def sm3(
   parameters; 2) adapts the learning rates in an adaptive and data-driven manner
   (like Adagrad and unlike Adafactor); and 3) comes with rigorous convergence
   guarantees in stochastic convex optimization settings.
-  
+
+  The algorithm initializes with a learning rate :math:`\eta`, maintains 
+  cumulative squared gradients :math:`\mu_t(r)` for each weight component 
+  :math:`r`, and updates :math:`w_t` using received gradients :math:`g_t`, 
+  adjusting each component based on the minimum accumulated gradient 
+  :math:`\nu_t(i)`. It employs a convention where division by zero yields zero, 
+  optimizing convergence by adapting to varying gradients across different 
+  parts of the weight vector.
+
+  SM3-I Algorithm
+
+  .. math::
```

Comment:
1. Use subscripts instead of parenthesizing unless it's a function.
2. Make the description follow the function implemented: here we implement SM3 as a gradient transformation, so the doc should detail the gradient transformation, not T iterations of the algorithm. You may detail then the `init_fn` and the `update_fn` (so that the initial values are described clearly).